app/src/main/res/values/strings.xml

```diff
@@ -137,7 +137,7 @@
     <string name="settings_hide_app_title">Hide the Magisk app</string>
     <string name="settings_hide_app_summary">Install a proxy app with random package ID and custom app label</string>
     <string name="settings_restore_app_title">Restore the Magisk app</string>
-    <string name="settings_restore_app_summary">Un-hide the app and restore it back to the original APK</string>
+    <string name="settings_restore_app_summary">Un-hide the app and change it back to its original APK</string>
```

Comment:
```suggestion
    <string name="settings_restore_app_summary">Un-hide the app by restoring it back to its original APK</string>
```

Reply:
Looks good!

Reply:
Wait, this one should probably end up matching the "Please restore it to the original APK." one we're workshopping below.

Reply:
So to match my other proposal above this should be:
```
Un-hide the app and restore the original APK
```

Reply:
I understand it as "Restores the original APK state, which un-hides the app." (in that order) which I think would mean:
"Resets the app, which will un-hide it."
How "original APK" has to come in confuses me.

Reply:
Because the hidden apk is modified, so restoring it uses the original again. I believe I already explained this to you.

Reply:
I couldn't find it, but now I understand. It is much less of an issue here.